src/dashboard/src/components/api/views.py

```diff
@@ -508,6 +485,8 @@ def copy_metadata_files_api(request):
     paths = request.POST.getlist('source_paths[]')
     return filesystem_ajax_views.copy_metadata_files(sip_uuid, paths)
 
+# TODO should this have auth?
+@_api_endpoint(expected_methods=['GET'])
 def get_levels_of_description(request):
```

Comment:
If so (for posterity), changes to dashboard/src/media/js/ingest/directory-metadata-form-dialog.js would be required.